ui/app/scripts/analytics/controllers/dashboard.js

```diff
@@ -101,32 +105,32 @@ define([], function () {
                     service: $injector.get('CounterService'),
                     streams: [],
                     visualizations: ['Bar-Chart', 'Graph-Chart'],
-                },
-                {
-                    label: 'Aggregate-Counters',
-                    service: $injector.get('AggregateCounterService'),
-                    streams: [],
-                    visualizations: ['Bar-Chart'],
-                    requestOptions: {resolution: 'minute'}
-                },
-                {
-                    label: 'Field-Value-Counters',
-                    service: $injector.get('FieldValueCounterService'),
-                    streams: [],
-                    visualizations: ['Bubble-Chart', 'Pie-Chart']
-                },
-                {
-                    label: 'Gauges',
-                    service: $injector.get('GaugeService'),
-                    streams: [],
-                    visualizations: ['Graph-Chart']
-                },
-                {
-                    label: 'Rich-Gauges',
-                    service: $injector.get('RichGaugeService'),
-                    streams: [],
-                    visualizations: ['Gauge-Chart']
-                }
+                }//,
```

Comment:
It is always better to delete the unwanted code rather than commenting out. We can always get back the code ;)